### ISSUE TITLE
The upload fails for the ds1 SBOM ubi9-9.3-782.json

### ISSUE THREAD

**Issue by PhilipCattanach** [trustification/trustify#816]
Running trustify locally in _PM Mode_.

Upload request for[ ubi9-9.3-782.json](https://github.com/trustification/trustification/blob/main/data/ds1/sbom/ubi9-9.3-782.json.bz2)  failed with a status code of 400

The other SBOMs within the ds1 dataset appear to have been uploaded successfully
Screenshot and the extract from the log are attached.
![UploadFailure](https://github.com/user-attachments/assets/6baac90f-4087-4cbc-9d89-9267c21aba67)
[TrustifyLogUploadError.txt](https://github.com/user-attachments/files/17030686/TrustifyLogUploadError.txt)

**Comment by helio-frota** [comment]
![2024-09-17_12-38](https://github.com/user-attachments/assets/f33b4fdd-c190-4f89-972b-6c43c1b728d9)

**Comment by helio-frota** [comment]
/me download the log file and not able to find the 400 error :man_shrugging: 

```console
➜  Downloads rg "200 " TrustifyLogUploadError.txt
1:2024-09-17T15:09:25.624921Z  INFO actix_web::middleware::logger: ::1 "GET /api/v1/sbom?limit=10&offset=0&sort=published:asc HTTP/1.1" 200 3115 "http://localhost:8080/sboms" "Mozilla/5.0 (X11; Linux x86_64; rv:130.0) Gecko/20100101 Firefox/130.0" 0.045210
47:2024-09-17T15:09:28.387769Z  INFO actix_web::middleware::logger: ::1 "GET /api/v1/sbom?limit=10&offset=0&sort=published:asc HTTP/1.1" 200 3545 "http://localhost:8080/sboms" "Mozilla/5.0 (X11; Linux x86_64; rv:130.0) Gecko/20100101 Firefox/130.0" 0.054015
48:2024-09-17T15:09:28.569723Z  INFO actix_web::middleware::logger: ::1 "GET /api/v1/sbom/urn:uuid:01920088-07db-7ae1-8741-f80fc6fd4324/packages?limit=1&offset=0 HTTP/1.1" 200 297 "http://localhost:8080/sboms" "Mozilla/5.0 (X11; Linux x86_64; rv:130.0) Gecko/20100101 Firefox/130.0" 0.125631
➜  Downloads rg "400 " TrustifyLogUploadError.txt
➜  Downloads
```

**Comment by ctron** [comment]
I just tried to upload ds1, and it works for me:

```
"spdx/ubi9-9.3-782.json.bz2": {
  "document_id": "https://access.redhat.com/security/data/sbom/beta/spdx/ubi9-container-f8098ef8-eee0-4ee6-b5d1-b00d992adef5",
  "id": "urn:uuid:019203e8-d12a-7721-932f-4362c62c4013"
},
```

And according to the uploaded log, it succeeds for you as well:

```
2024-09-17T15:09:28.328983Z  INFO actix_web::middleware::logger: ::1 "POST /api/v1/sbom HTTP/1.1" 201 138 "http://localhost:8080/sboms" "Mozilla/5.0 (X11; Linux x86_64; rv:130.0) Gecko/20100101 Firefox/130.0" 0.524177    
```

@carlosthe19916 Could that be a UI issue?

**Comment by ctron** [comment]
Just tried the same way with the UI: 
![image](https://github.com/user-attachments/assets/e1256324-f406-4528-aa37-ec7f168a939c)

**Comment by ctron** [comment]
Ok, I created a unit test, uploading multiple sboms in parallel and can see the "deadlock" issue then too. I assume that's the case there as well, as the document itself ingests fine.

Duplicate of: https://github.com/trustification/trustify/issues/817

**Comment by ctron** [comment]
Duplicate of: https://github.com/trustification/trustify/issues/817
